modules/aws/ecr.go

```diff
@@ -0,0 +1,93 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecr"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateECRRepo creates a new ECR Repository
```

Comment:
NIT: We've been adding a note on the non-E functions to mention that:
```suggestion
// CreateECRRepo creates a new ECR Repository. This will fail the test and stop execution if there is an error.
```

This applies to all the non-E functions in this file.

Reply:
Done in 2840220